### PR TITLE
Play local file on UWP and Android adjustments and MediaExtractor Refactor

### PR DESCRIPTION
**Pull request by orzech85**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
feature/fix

### :arrow_heading_down: What is the current behavior?
- local files are not loaded correctly

### :new: What is the new behavior (if this is a feature change)?
- local files are loaded on UWP and Android
- MediaExtractors is a bit cleaned (shared code is extracted to the base class)

### :boom: Does this PR introduce a breaking change?
no

### :thinking: Checklist before submitting

- [x] All projects build
- [ ] Follows style guide lines 
- [x] Relevant documentation was updated
- [x] Rebased onto current develop

### CODE REVIEW

File: MediaManager/Platforms/Ios/Video/VideoView.cs [hunk 1]
```diff
@@ -108,13 +108,5 @@ public VideoAspectMode VideoAspect
             get => PlayerViewController.ShowsPlaybackControls;
             set => PlayerViewController.ShowsPlaybackControls = value;
         }
-
-        protected override void Dispose(bool disposing)
```
Comment (martijn00) [inline MediaManager/Platforms/Ios/Video/VideoView.cs]: This should not be removed?

Comment (orzech85) [reply MediaManager/Platforms/Ios/Video/VideoView.cs]: It caused a crash after a few videos play in Xamarin.Forms. In native is not called at all.

Comment (martijn00) [reply MediaManager/Platforms/Ios/Video/VideoView.cs]: Ok, but then maybe we need to move this function to some different method.

Comment (orzech85) [reply MediaManager/Platforms/Ios/Video/VideoView.cs]: Do you have any idea? What was the purpose of that?

Comment (orzech85) [reply MediaManager/Platforms/Ios/Video/VideoView.cs]: I moved it to the `PlayerViewController`

Comment (orzech85) [reply MediaManager/Platforms/Ios/Video/VideoView.cs]: I moved it to `PlayerViewController`,

**Pull request merged**